libcxx/test/std/atomics/types.pass.cpp

```diff
@@ -17,6 +17,9 @@
 //     typedef T value_type;
 // };
 
+// atomic still has a difference_type in the C++03 frozen headers
+// XFAIL: FROZEN-CXX03-HEADERS-FIXME
```

Comment:
Thanks for helping with the CI.
Do you know why we test with C++03 in the first place?  `std::atomic` was introduced in C++11